doc/sphinx-guides/source/admin/integrations.rst

```diff
@@ -95,6 +111,17 @@ Researchers can launch Jupyter Notebooks, RStudio, and other computational envir
 
 Institutions can self host BinderHub. Dataverse is one of the supported `repository providers <https://binderhub.readthedocs.io/en/latest/developer/repoproviders.html#supported-repoproviders>`_.
 
+Renku
++++++
+
+Researchers can import Dataverse datasets into their Renku projects via the
+command-line interface (CLI) by using the Dataverse DOI. See the `renku Dataset
```

Comment:
A Dataverse installation can run handles as well, but I'm OK with this as is. We can revisit as we'll be working in this area quite a bit over the coming months.

Reply:
actually I think what we do is either process the DOI or extract it from a dataverse URL. Are either of those possibly incompatible with how some installations might format URLs? Maybe @Panaetius can chime in about our implementation.

Reply:
An example of an installation running handles is:

http://hdl.handle.net/11529/10016
https://data.cimmyt.org/dataset.xhtml?persistentId=hdl:11529/10016

Reply:
right, I remember some discussion about that - I think at the moment we require a DOI for import. I also don't see an option to export JSON-LD on that installation?

Reply:
so should I qualify the statement in the docs to be clearer about the current requirements?

Reply:
Yeah, they are on 4.7.1 and it's been added since then (4.8.x)

Reply:
Yes, it would be good to add some more details. Thanks!

Reply:
There's 3 way that work, either just a plain DOI, or a DOI that's prefixed with `doi:` or an URL that has the following conditions:
- it contains a `persistentId` entry
- it has the dataverse version page at the `api/v1/info/version` path
 in which case we access `<domain>/api/v1/datasets/export/?persistentId=<persistentId>&exporter=schema.org` to get the data.

So short answer, handles also work. Long answer, I remember some dataverse instances that use handles being weird as they don't have the API enabled or don't use `persistentId`. But it's been a while since I did this.

In your examples, the first URL wouldn't work, the second should work (though the first just redirects to the second).

Reply:
I tried both, neither seemed to work.